app/src/main/java/org/simple/clinic/bp/sync/BloodPressureMeasurementPayload.kt

```diff
@@ -59,4 +59,21 @@ data class BloodPressureMeasurementPayload(
         //FIXME: [Encounter] This code is throw-away. Once the Encounter API is implemented, this should be fixed.
         encounterUuid = generateEncounterUuid(facilityUuid, patientUuid, recordedAt.toLocalDateAtZone(ZoneOffset.UTC)))
   }
+
+  fun toDatabaseModel(syncStatus: SyncStatus, encounterUuid: UUID): BloodPressureMeasurement {
```

Comment:
Make this function slightly more descriptive. So that maintainers don't have to look at the function's return type to figure out what it is doing?

Reply:
This method is kind of a standard throughout the app for all our `Payload` classes.
So if I change the name or signature here, I will have to change it in other places as well.

Its `Payload.toDatabaseModel` which I feel is explicit enough. Can you suggest what should I change this to?

Reply:
If this is convention, I am probably not used to this. It is fine then.